src/Eccube/Resource/template/default/Shopping/shipping_edit.twig

```diff
@@ -35,7 +35,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% endblock javascript %}
 
 {% block main %}
-    <h1 class="page-heading">お届け先の追加</h1>
+    <h1 class="page-heading">
+    {% if app.user is not null and is_granted('ROLE_USER') %}
```

Comment:
Because of some internal Symfony details, to avoid broken error pages in the prod environment, wrap calls in these templates with a check for app.user
Ref: 
http://symfony.com/doc/2.7/security.html#access-control-in-templates
http://symfony.com/blog/new-in-symfony-2-8-dx-improvements#allow-to-check-for-security-even-in-pages-not-covered-by-firewalls